test/powershell/Modules/Microsoft.WSMan.Management/CredSSP-nonAdmin.Tests.ps1

```diff
@@ -0,0 +1,27 @@
+Describe "CredSSP cmdlet tests non-admin" -Tags 'Feature' {
+
+    $isAdmin = (New-Object Security.Principal.WindowsPrincipal ([Security.Principal.WindowsIdentity]::GetCurrent())).IsInRole([Security.Principal.WindowsBuiltinRole]::Administrator)

```

Comment:
1. will wrap in $IsWindows
2. will wrap in BeforeAll
3. this is the opposite of requiring admin, this test requires NOT admin
4. two different test cases here where the first is only applicable when NOT admin, the second one works either way